test/ValidateID.cpp

```diff
@@ -821,6 +821,53 @@ TEST_F(ValidateID, OpStoreTypeBad) {
   CHECK(spirv, SPV_ERROR_INVALID_ID);
 }
 
+TEST_F(ValidateID, OpStoreVoid) {
```

Comment:
No, they shouldn't.  Not only is this convention sparsely used (less than half of the total test names, even counting those with the _prefix_ instead of the suffix), it adds no useful information and provides a false sense of coverage.

Before 276a724b2580dc66936b5235f672c03241937cff, `OpTypeVectorGood` didn't say whether it was testing an int, float, or bool vector -- it was just letting us know that it was, well, "good".  Similarly, before 961f5dc54408b1516b9e4d85ee1fb4891b903f61, `OpExecutionModeEntryPointBad` was, in fact, testing a _missing_ entry point, not a bad one.  So the "Good/Bad" suffix says nothing about what the test actually covers.  When such a test fails, the name doesn't suggest which functionality failed.  A good name would tell me immediately that, eg, a Boolean vector failed while the int one passed.

Most of the FooGood/FooBad tests leave out lots of coverage; they seem to forget there's more than one good or bad case.  Over the past week, I've been working hard to improve that.

Furthermore, once we move away from the `CHECK` macro to a more nuanced way of testing the validation outcome (hello, `ValidateFixtures` :), the distinguishing power of the "Bad" suffix will further erode: does "Bad" mean failed parsing, failed validation, one of several non-success statuses, a particular error message, or what?  It's not clear, and therefore not useful.